src/core/sagas/settings.ts

```diff
@@ -56,7 +56,7 @@ function* getPlexLoginUrl() {
 }
 
 function* getSettings() {
-  yield put(startFetching('settings'));
+  // yield put(startFetching('settings'));
```

Comment:
should this be commented out ?

Reply:
Yes, I did some "magic" there.
So that when you save a setting, the spinner doesn't show up.
I'll have to rewrite the code there to be more understandable.